developer-docs-site/docs/concepts/staking.md

```diff
@@ -110,10 +109,6 @@ You must stake the required minimum amount to join the validator set. Moreover,
 
 If at any time after joining the validator set, your current staked amount exceeds the maximum allowed stake (for example as the rewards are added to your staked amount), then your voting power and the rewards will be calculated only using the maximum allowed stake amount, and not your current staked amount. 
 
-:::tip When the staked amount falls below minimum
```

Comment:
why did we remove this?

Reply:
This is a theoretical point, because the staked amount doesn't fall below minimum unless we apply slashing. And we don't have slashing feature yet. I've been meaning to remove this from earlier version also.

Reply:
The owner could technically withdraw part of the stake out and leave their balance below the min required. In that case, their stake pool will get kicked out of the validator set when the next epoch starts.